src/pages/messagebox/index.tsx

```diff
@@ -1,43 +1,63 @@
 import { css } from '@emotion/react'
 import styled from '@emotion/styled'
 import { Modal, useMedia } from '@offer-ui/react'
+import type { GetServerSideProps } from 'next'
+import { useRouter } from 'next/router'
 import { useState, type ReactElement, useEffect } from 'react'
+import { toQueryString } from '@utils/format'
+import { useGetMessageRooms } from '@apis'
 import {
   MessagePreview,
   Tabs,
   Tab,
   ChattingRoom,
   MessageBoxPlaceholder
 } from '@components'
-import { IMAGE } from '@constants'
+import { IMAGE, MESSAGE_SORT_OPTIONS } from '@constants'
 import { useModal } from '@hooks'
+import type { MessageSortTypeCodes } from '@types'
 
-type TabType = 'all' | 'buy' | 'sell'
-
-const TABS = {
-  all: '전체',
-  buy: '구매',
-  sell: '판매'
-} as const
-
-const TabKeys = Object.keys(TABS) as TabType[]
-const TabEntries = Object.entries<TabType, ValueOf<typeof TABS>>(TABS)
+type RoomId = number | null
+type Props = {
+  roomId: RoomId
+}
 
-const MessageBoxPage = (): ReactElement => {
-  const [tab, setTab] = useState<TabType>('all')
-  const [roomId, setRoomId] = useState<number | null>(null)
+export const getServerSideProps: GetServerSideProps<Props> = async ({
+  query
+}) => ({
+  props: {
+    roomId: query.roomId ? Number(query.roomId) : null
+  }
+})
+
+const MessageBoxPage = ({ roomId: defaultRoomId }: Props): ReactElement => {
+  const [tab, setTab] = useState<MessageSortTypeCodes>('ALL')
```

Comment:
```suggestion
  const [sortType, setSortType] = useState<MessageSortTypeCodes>('ALL')
```
tab대신 sortType으로 하면 더 직관적일 것 같습니다!

Reply:
[변경 완료 ! ](https://github.com/price-offer/offer-fe/pull/196/commits/d0fb4bb7b5177f6042d31d2e6486d65b49fac2c7#diff-1efeb46e8822d73803df6f18f0f03a7bf7e285c0c3d2aaae69bb0630658519ceR34)